amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpSink.scala

```diff
@@ -6,46 +6,40 @@ package akka.stream.alpakka.amqp.scaladsl
 
 import akka.Done
 import akka.stream.alpakka.amqp._
-import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.{Keep, Sink}
 import akka.util.ByteString
 
 import scala.concurrent.Future
 
 object AmqpSink {
 
   /**
-   * Scala API: Creates an [[AmqpSink]] that accepts ByteString elements.
+   * Creates an [[AmqpSink]] that accepts [[WriteMessage]] elements.
```

Comment:
Scaladoc requires fully-qualified class names even if the class is imported.

Reply:
I changed it to backticks, as you recommended for AmqpFlow in one of the previous comments.